GooglePayment/src/main/java/com/braintreepayments/api/models/ReadyForGooglePayRequest.java

```diff
@@ -0,0 +1,32 @@
+package com.braintreepayments.api.models;
+
+import com.braintreepayments.api.BraintreeFragment;
+import com.braintreepayments.api.GooglePayment;
+import com.braintreepayments.api.interfaces.BraintreeResponseListener;
+
+/**
+ * Optional parameters to use when checking whether Google Payment API is supported and set up on the device.
+ */
+public class ReadyForGooglePayRequest {
+
+    private boolean mExistingPaymentMethodRequired;
+
+    /**
+     * If set to true, then the {@link GooglePayment#isReadyToPay(BraintreeFragment, ReadyForGooglePayRequest, BraintreeResponseListener)}
+     * method will call the listener with true if the customer is ready to pay with one or more of your
+     * supported card networks.
+     *
+     * @param value Indicates whether the customer must have one or more payment methods from your allowed card networks in order
+     *              to be considered ready to pay with Google Pay
+     *
+     * @return {@link ReadyForGooglePayRequest}
+     */
+    public ReadyForGooglePayRequest existingPaymentMethodRequired(boolean value) {
```

Comment:
In the rest of the android SDK, I think we pass the actual variable name instead of `value` here. But this name is super long, so I understand why you didn't.

Not sure if we'd rather be consistent.